packages/flutter/lib/src/widgets/overlay.dart

```diff
@@ -138,6 +139,24 @@ class OverlayEntry implements Listenable {
     _overlay!._didChangeEntryOpacity();
   }
 
+  /// Whether the content of this [OverlayEntry] can be used to size the
+  /// [Overlay].
+  ///
+  /// In most situations the Overlay sizes itself based on its incoming
+  /// constraints to be as large as possible. However, if that would result in
+  /// an infinite size, it has to rely on one of its children to size itself. In
+  /// this situation, the Overlay will consult the topmost [OverlayEntry] that
+  /// has this property set to true, lay it out with unconstrained
+  /// [BoxConstraints], and force all other OverlayEntries to have the same
+  /// size.
+  ///
+  /// OverlayEntries that set this to true must be able to handle unconstrained
+  /// [BoxConstraints].
+  ///
+  /// Setting this to true has no effect if the OverlayEntry uses a [Positioned]
+  /// widget to position itself in the Overlay.
```

Comment:
nit: it's pretty clear what "Overlay" and "OverlayEntries" are referring to, so I'd just use the plain English phrases "overlay" and "overlay entries" to refer to them. Otherwise, without the `[...]` they are kind of in limbo state, neither proper symbol links, nor proper nouns.

Reply:
good point, updated.